client/src/pages/privacy.md

```diff
@@ -0,0 +1,216 @@
+
+## Storage and security of personal information 
+
+This site is hosted and stored in Australia in secure, government-accredited facilities. To help protect the privacy of the data and personal information we collect and hold, we maintain physical, technical and administrative safeguards. 
+
+Access to your personal information is restricted to DTA employees working on the oneAPS Opportunities pilot who need it to provide services to you and to the Opportunity creator who is seeking a person to fulfill their opportunity. We take steps to protect the security of the personal information we hold from both internal and external threats by: 
+
+* regularly assessing the risk of misuse, interference, loss, and unauthorised access, modification or disclosure of that information 
+
+* taking measures to address those risks, for example, we keep a record (audit trail) of when someone has added, changed or deleted personal information held in our electronic databases and regularly check that staff only access those records when they need to 
+
+* conducting regular internal and external audits to assess whether we have adequately complied with or implemented these measures. 
+
+## Accessing and correcting your personal information 
+
+Under the Privacy Act (Australian Privacy Principles 12 and 13) you have the right to ask for access to personal information that we hold about you and ask that we correct that personal information.  
+
+You can ask for access or correction by contacting us and we must respond within 30 days. If you ask, we must give you access to your personal information, and take reasonable steps to correct it if we consider it is incorrect, unless there is a law that allows or requires us not to. 
+
+We will ask you to verify your identity before we give you access to your information or correct it, and we will try to make the process as simple as possible. If we refuse to give you access to, or correct, your personal information, we must notify you in writing setting out the reasons. 
+
+If we make a correction and we have disclosed the incorrect information to others, you can ask us to tell them about the correction. We must do so unless there is a valid reason not to. 
+
+If we refuse to correct your personal information, you can ask us to associate with it (for example, attach or link) a statement that you believe the information is incorrect and why. 
+
+You also have the right under the Freedom of Information Act 1982 to request access to documents that we hold and ask for information that we hold about you to be changed or annotated if it is incomplete, incorrect, out-of- date or misleading.   
+
+## How to contact us 
+
+Requests for access to your own personal information are free. 
+
+Request for your own personal information may be made to: 
+
+**Email** 
+<a href="mailto:foi@dta.gov.au">foi@dta.gov.au</a>
+   
+**Post** 
+FOI Officer 
+Digital Transformation Agency 
+PO Box 457 
+Canberra City 
+ACT 2601 
```

Comment:
Do we want this address formatted on one line?  It might be easier to read if we break it up.

This is what it looks like currently:

<img width="627" alt="Screen Shot 2021-07-06 at 10 22 19 am" src="https://user-images.githubusercontent.com/2645932/124525861-30a20900-de44-11eb-8ee5-856da92102a2.png">

With changes applied:

<img width="229" alt="Screen Shot 2021-07-06 at 10 24 04 am" src="https://user-images.githubusercontent.com/2645932/124525912-5e874d80-de44-11eb-80e2-da88ba287fb3.png">


```suggestion
**Post**  
FOI Officer  
Digital Transformation Agency  
PO Box 457  
Canberra City 
ACT 2601
```